crates/router/src/services/api.rs

```diff
@@ -1258,6 +1263,71 @@ pub fn build_redirection_form(
+                        })
+                        </script>
+                        "#))
+
+
+                                     h3 style="text-align: center;" { "Please wait while we process your payment..." }
+                                 }
+
+             (PreEscaped(format!(
+                 r#"
+                    <script type="module">
+                        import kountSDK from 'https://cdn.jsdelivr.net/gh/Kount/kount-web-sdk/kount-web-client-sdk.js';
+                        document.addEventListener('DOMContentLoaded', function() {{
+                            const sessionID = "{}";
+                            const kountConfig = {{
+                                clientID: "{}",
+                                environment: "{}",
+                                isSinglePageApp: true,
+                                isDebugEnabled: true
+                            }};
+                            const sdk = new kountSDK(kountConfig, sessionID);
+                        }});
+                        var f = document.createElement('form');
+                        f.action = window.location.pathname.replace(new RegExp("payments/ddc/(\\w+)/(\\w+)"), "payments/complete_ddc/$1/$2");
+                        f.method='POST';
+                        document.body.appendChild(f);
+                        f.submit();
```

Comment:
This block will get executed even before the Kount SDK call. Add this within `DOMContentLoaded` block and execute with SetTimeout so that we can be sure that kountSDK is called before redirection
```suggestion
                        var f = document.createElement('form');
                        f.action = window.location.pathname.replace(new RegExp("payments/ddc/(\\w+)/(\\w+)"), "payments/complete_ddc/$1/$2");
                        f.method='POST';
                        document.body.appendChild(f);
                        f.submit();
```